Dockerfile

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
looks like a bad merge here - would a `make build` have caught this?

Reply:
Thanks for spotting this. Also no, `make build` doesn't catch this.